gammapy/modeling/scipy.py

```diff
@@ -69,14 +69,11 @@ def _confidence_scipy_brentq(
     )
 
     lower_bound = parameter.factor
-    upper_bound = parameter.factor_max if upper else parameter.factor_min
-    if np.isnan(upper_bound):
-        # TODO: remove hard coded limits here...
-        upper_bound = parameter.factor
-        if upper:
-            upper_bound += 1e2 * parameter.error / parameter.scale
-        else:
-            upper_bound -= 1e2 * parameter.error / parameter.scale
+
+    if upper:
```

Comment:
Why removing the initial condition? It there is a finite range defined it seems fair to stick to it no?

Reply:
In particular, the error message below suggests to change the parameter boundaries.

Reply:
The difference is that the parameter min/max value are not necessarily defined, while the `scan_min` / `scan_max` are derived from the parameter errors by default. But indeed one could implement the order like:
- check `par.min` and `par.max` first, if those are defined use those
- if not check `scan_min` and `scan_max` and use it 
- throw an error else...

Reply:
For flux points estimation there is currently no access to the norm fit boundaries either...